src/rocksdb.rs

```diff
@@ -2590,4 +2631,33 @@ mod test {
         let cf_opts = db.get_options_cf(cf);
         assert_eq!(cf_opts.get_disable_auto_compactions(), true);
     }
+
+    #[test]
+    fn test_load_latest_options() {
+        let path = TempDir::new("_rust_rocksdb_load_latest_option").expect("");
+        let dbpath = path.path().to_str().unwrap().clone();
+        let cf_name: &str = "dorianzheng";
+
```

Comment:
Can you add a test when the db does not exist?

Reply:
Good idea

Reply:
use a better name instead of "dorianzheng"